bgpd/bgp_route.c

```diff
@@ -9729,8 +9729,13 @@ void route_vty_out_tmp(struct vty *vty, struct bgp_dest *dest,
 		}
 	}
 	if (use_json) {
+		struct bgp_path_info *bpi = bgp_dest_get_bgp_path_info(dest);
+
 		json_object_boolean_true_add(json_status, "*");
 		json_object_boolean_true_add(json_status, ">");
+
+		if (CHECK_FLAG(bpi->flags, BGP_PATH_MULTIPATH))
```

Comment:
blech. blech?  call the flag multiPath?

Reply:
I just kept the same format as others. Or I can do this with another commit, but for all flags, not only this. Does it work?